server/src/main/root/META-INF/context.xml

```diff
@@ -1,2 +1,6 @@
 <!-- Testing repositories have to contain symlinks, when golden ticket is used -->
-<Context override="true" allowLinking="true" />
+<Context override="true" allowLinking="true">
```

Comment:
Quick question about the file paths here.  I see we have it under `src/main/root` and `src/main/webapp`.  I've never seen `src/main/root` used before.  Would you mind elaborating a little on what this path does?

Reply:
I was not sure if this change is needed in both context files so I did a bit more testing. I seems `src/main/root` is not used for Candlepin but for setup of testing repository. Only change in webapp should be needed.

Reply:
Okay.  If you want to back this file out then, that's fine.

Reply:
Reverted changes to this file.